packages/suite/src/views/guide/Feedback.tsx

```diff
@@ -81,10 +81,7 @@ const RatingItem = styled.button<{ selected?: boolean }>`
     border: 1px solid ${props => props.theme.STROKE_GREY};
     cursor: pointer;
     font-size: 30px;
-
-    &:hover {
-        background: ${props => props.theme.STROKE_GREY};
-    }
+    background-color: inherit;
```

Comment:
Is removing the hover effect really intended? The emoji buttons feels kinda 'dead' without it.

Reply:
This is from Zeplin
![Screenshot 2021-06-28 at 18 18 09](https://user-images.githubusercontent.com/33235762/123670163-35136400-d83d-11eb-9382-2317336320ad.png)

Reply:
ok.

Reply:
I am thinking about it again and it is the only place where hover effect is not available, I will put it there back.